wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -175,6 +175,12 @@ def get_status(self) -> Status:
             return_status = Status("failed")
         elif status.active == 1:
             return Status("running")
+        elif status.conditions is not None:
```

Comment:
On older versions of kubernetes does the `conditions` attribute always exist?

Reply:
Looks like it was added in v1.23. I'll add a check to make sure it's there